services/QuillLMS/app/controllers/api/v1/app_settings_controller.rb

```diff
@@ -0,0 +1,17 @@
+class API::V1::AppSettingsController < ApplicationController
+  def index
```

Comment:
On the fence regarding authentication in this controller - share your thoughts.

Reply:
I feel like authentication is something that needs to be addressed, but probably out of scope for this feature.